libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid-wrapper.component.ts

```diff
@@ -220,4 +254,59 @@ export class SkyAgGridWrapperComponent
       }
     }
   }
+
+  #updateInlineHelp(column: Column, context?: any): void {
+    const colId = column.getId();
+    const columnHeader = this.elementRef.nativeElement.querySelector(
+      `div.ag-header-cell[col-id="${colId}"] div.ag-cell-label-container`
+    ) as HTMLElement;
+    const inlineHelpComponent: Type<unknown> | undefined =
+      column.getColDef().headerComponentParams?.inlineHelpComponent;
+    const currentComponent: ComponentRef<unknown> | undefined =
+      this.#columnInlineHelpComponents.get(colId);
+
+    const getHostElement = (): HTMLElement => {
+      if (!columnHeader.querySelector(`span.sky-control-help-container`)) {
+        const inlineHelpContainer = this.document.createElement('span');
+        inlineHelpContainer.classList.add('sky-control-help-container');
+        columnHeader.appendChild(inlineHelpContainer);
+      }
+      return columnHeader.querySelector(`span.sky-control-help-container`);
+    };
+
+    if (inlineHelpComponent) {
+      const referenceEl = getHostElement();
+      if (currentComponent) {
+        if (
+          currentComponent.componentType !== inlineHelpComponent ||
+          referenceEl.firstElementChild !==
+            currentComponent.location.nativeElement
+        ) {
+          currentComponent.destroy();
```

Comment:
Updating a header. The element check was necessary because AG Grid destroys the container element.